src/test/resources/AddSubtract.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="AddSubtract">
+    <test verbose="2" preserve-order="true" name="C:\Users\EVA Unit-11\Desktop\EPAM\selenium\HW1. Unit Frameworks\HW1">
```

Comment:
Why such a strange name for a test?
`name="C:\Users\EVA Unit-11\Desktop\EPAM\selenium\HW1. Unit Frameworks\HW1"`